clio-server/src/test/scala/org/broadinstitute/clio/server/service/WgsCramServiceSpec.scala

```diff
@@ -53,6 +59,8 @@ class WgsCramServiceSpec extends TestKitSuite("WgsCramServiceSpec") {
         Seq(
           WgsCramService.v1QueryConverter.buildQuery(
             transferInput.copy(documentStatus = Option(DocumentStatus.Normal))
+          )(
+            ElasticsearchIndex.WgsCram
```

Comment:
Doens't need newline

Reply:
And yet sbt thinks it does.

Reply:
Hwot

Reply:
I can try to move things around and see if I can get the autoformatter to not ruin everything